languages/smaily-for-contact-form-7-en.po

```diff
@@ -0,0 +1,167 @@
+
+#: admin/class-smaily-for-cf7-admin.php:284
+msgid "Please fill out all fields!"
+msgstr "Please fill out all fields!"
+
+#: admin/class-smaily-for-cf7-admin.php:317
+msgid "Credentials removed"
+msgstr "Credentials removed"
+
+#: admin/class-smaily-for-cf7-admin.php:320
+msgid "No credentials to remove"
+msgstr "No credentials to remove"
+
+#: admin/partials/smaily-for-cf7-admin-display.php:14
+msgid ""
+"Configuring Smaily integration is disabled when using \"Add New Form\". "
+"Please save this form or edit an already existing form"
+msgstr ""
+"Configuring Smaily integration is disabled when using \"Add New Form\". "
+"Please save this form or edit an already existing form"
+
+#: admin/partials/smaily-for-cf7-admin-display.php:22
+msgid "Your API credentials are valid"
+msgstr "Your API credentials are valid"
+
+#: admin/partials/smaily-for-cf7-admin-display.php:25
+msgid "Autoresponder"
+msgstr "Autoresponder"
+
+#: admin/partials/smaily-for-cf7-admin-display.php:28
+msgid "No autoresponder"
+msgstr "No autoresponder"
+
+#: admin/partials/smaily-for-cf7-admin-display.php:42
+msgid "Reset credentials"
+msgstr "Reset credentials"
+
+#: admin/partials/smaily-for-cf7-admin-display.php:46
+msgid "Connect form with Smaily"
+msgstr "Connect form with Smaily"
+
+#: admin/partials/smaily-for-cf7-admin-display.php:49
+msgid "Captcha disabled. Please use a captcha if this is a public site."
+msgstr "Captcha disabled. Please use a captcha if this is a public site."
+
+#: admin/partials/smaily-for-cf7-admin-display.php:53
+msgid "Saved credentials invalid."
+msgstr "Saved credentials invalid."
+
+#: admin/partials/smaily-for-cf7-admin-display.php:58
+msgid "Subdomain"
+msgstr "Subdomain"
+
+#: admin/partials/smaily-for-cf7-admin-display.php:68
+msgid "API Username"
+msgstr "API Username"
+
+#: admin/partials/smaily-for-cf7-admin-display.php:75
+msgid "API Password"
+msgstr "API Password"
+
+#: admin/partials/smaily-for-cf7-admin-display.php:88
+msgid "Verify credentials"
+msgstr "Verify credentials"
+
+#: public/class-smaily-for-cf7-public.php:61
+msgid ""
+"Smaily for CF7 requires Transliterator extension. Please install php-intl "
+"package and try again."
+msgstr ""
+"Smaily for CF7 requires Transliterator extension. Please install php-intl "
+"package and try again."
+
+#: public/class-smaily-for-cf7-public.php:181
+msgid "Form was not submitted using POST method."
+msgstr "Form was not submitted using POST method."
+
+#: public/class-smaily-for-cf7-public.php:184
+msgid "Input does not contain a valid email address."
+msgstr "Input does not contain a valid email address."
+
+#: public/class-smaily-for-cf7-public.php:187
+msgid "Could not add to subscriber list for an unknown reason."
+msgstr "Could not add to subscriber list for an unknown reason."
+
+#: smaily-for-contact-form-7.php:74
+msgid ""
+"Smaily for Contact Form 7 is not able to activate.\n"
+"\t\t\tContact Form 7 is needed to function properly. Is Contact Form 7 "
```

Comment:
Hm. Tabs (or excessive whitespace) in translation phrases is not good. Can you split it into multiple strings? Or if it is due to formatting, could you combine it from multiple strings - something like `__( "foo" . "bar", "text-domain" )`?